.devcontainer/README.md

```diff
@@ -0,0 +1,99 @@
+# Dev Container Setup for ComfyStream
+
+This guide will help you set up and run a development container for ComfyStream using Visual Studio Code (VS Code).
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/get-started)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [VS Code Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+
+## Building the Base Container
+
+To build the base container, run the following command in your terminal:
+
+```sh
+docker build -f .devcontainer/Dockerfile.base -t livepeer/comfyui-base:latest .
+```
+
+## Using the Pre-built Base Container
+
+Most users will configure host paths for models in `devcontainer.json` and use the pre-built base container. Follow these steps:
+
+1. Pull the pre-built base container:
+
+```sh
+docker pull livepeer/comfyui-base:latest
+```
+
+2. Configure the host paths for models in the `devcontainer.json` file.
+
+3. Re-open the workspace in the dev container using VS Code:
+    - Open the Command Palette (`Ctrl+Shift+P` or `Cmd+Shift+P` on macOS).
+    - Select `Remote-Containers: Reopen in Container`.
+
+## Configuration
+
+Create a directory to store the models
```

Comment:
What kind of models are you refering to here? ComfyUI model checkpoints?

Reply:
Yes, all types of models. Checkpoints, ControlNet, Onnx, etc. Models to download are defined in [models.yaml](https://github.com/eliteprox/comfystream/blob/6ba659c453ae06a7b030229be1d8f8a73cd0f3e3/src/comfystream/scripts/setup_models.py) and  downloaded by [setup_models.py](https://github.com/eliteprox/comfystream/blob/6ba659c453ae06a7b030229be1d8f8a73cd0f3e3/src/comfystream/scripts/setup_models.py).